relay-metrics/src/aggregation.rs

```diff
@@ -690,15 +690,10 @@ pub struct Bucket {
     pub timestamp: UnixTimestamp,
     /// The length of the time window in seconds.
     pub width: u64,
-    /// The name of the metric without its unit.
+    /// The MRI (metric resource identifier).
     ///
     /// See [`Metric::name`].
     pub name: String,
```

Comment:
Not part of the PR, but if the identifier of a bucket is going to be the MRI, I think we should rename this property.

Reply:
Unfortunately this is public-facing API so we'd have to add `#[serde(rename = "name")]` then